src/studio-home/card-item/CardItem.test.tsx

```diff
@@ -29,15 +29,17 @@ describe('<CardItem />', () => {
     render(<CardItem {...props} />);
     const courseTitleLink = screen.getByText(props.displayName);
     expect(courseTitleLink).toHaveAttribute('href', `${getConfig().STUDIO_BASE_URL}${props.url}`);
+    const dropDownMenu = screen.getByTestId('toggle-dropdown');
```

Comment:
Please try to [avoid `getByTestId`](https://testing-library.com/docs/queries/about#priority). In fact, I can see there's an accessibility issue with this button - it doesn't have a `name` for non-visual users
![Screenshot](https://github.com/user-attachments/assets/4dc83470-166e-4cd8-beaa-a0a44538c9a3)

![Screenshot](https://github.com/user-attachments/assets/f24711e4-1d68-45e0-bdbf-8b0cf5a42ad1)

Please add an `aria-label` to the button then change this to something like `screen.getByRole('button', {name: /Course Actions/i})`